<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 117, in _parse_yaml
    output = cast(JsonType, yaml.safe_load(s))
                            ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/composer.py", line 35, in get_single_node
    if not self.check_event(StreamEndEvent):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 142, in parse_implicit_document_start
    if not self.check_token(DirectiveToken, DocumentStartToken,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 258, in fetch_more_tokens
    raise ScannerError("while scanning for the next token", None,
yaml.scanner.ScannerError: while scanning for the next token
found character '%' that cannot start any token
  in "<unicode string>", line 1, column 2:
    {% set tests_to_skip = tests_to_s ...
     ^

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 728, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 834, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 894, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_syntax.py", line 35, in check_recipe
    ro_parser: Final[RecipeParser] = recipe.get_read_only_parser()
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 675, in get_read_only_parser
    return RecipeParser("\n".join(self.meta_yaml))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 328, in __init__
    new_node = RecipeParser._parse_line_node(clean_line)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 147, in _parse_line_node
    output = RecipeParser._parse_yaml(s)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 126, in _parse_yaml
    cast(JsonType, yaml.safe_load(s)), lambda d: substitute_markers(d, sub_list)
                   ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/composer.py", line 35, in get_single_node
    if not self.check_event(StreamEndEvent):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 142, in parse_implicit_document_start
    if not self.check_token(DirectiveToken, DocumentStartToken,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 258, in fetch_more_tokens
    raise ScannerError("while scanning for the next token", None,
yaml.scanner.ScannerError: while scanning for the next token
found character '%' that cannot start any token
  in "<unicode string>", line 1, column 2:
    {% set tests_to_skip = tests_to_s ...
     ^

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_6ap9hsx2m9/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -39,6 +39,14 @@ requirements:
 {% set tests_to_skip = tests_to_skip + " test_getitems_when_iter_with_null" %}  # [py<39]
 {% set tests_to_skip = tests_to_skip + " or test_to_numpy" %}                   # [py<39]
 
+# ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
+{% set tests_to_skip = tests_to_skip + " or "test_date_max_2d" %}               # [s390x]
+{% set tests_to_skip = tests_to_skip + " or "test_date_min_2d" %}               # [s390x]
+{% set tests_to_skip = tests_to_skip + " or "test_date_median_2d" %}            # [s390x]
+# AssertionError: assert array(...)
+{% set tests_to_skip = tests_to_skip + " or "test_asdatetime %}                 # [s390x]
+
+
 # pandas 2.0.3 (last py38 build) really doesn't like the JSON
 # compliance tests, 325 of them
 {% set tests_to_ignore = "" %}

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -8,7 +8,7 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/db_dtypes-{{ version }}.tar.gz
   sha256: 7bcbc8858b07474dc85b77bb2f3ae488978d1336f5ea73b58c39d9118bc3e91b
-  patches:
+  patches:                                                         # [py<39]
     # pandas 2.0.3 (last py38 build) does not support BaseReduceTests
     - patches/0001-remove-test-against-base.BaseReduceTests.patch  # [py<39]
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,15 +36,15 @@ requirements:
 # pandas 2.0.3 (last py38 build) doesn't support this attribute
 {% set tests_to_skip = "" %}
 # AttributeError: 'JSONArray' object has no attribute '_data'
-{% set tests_to_skip = tests_to_skip + " test_getitems_when_iter_with_null" %}  # [py<39]
+{% set tests_to_skip = tests_to_skip + "test_getitems_when_iter_with_null" %}   # [py<39]
 {% set tests_to_skip = tests_to_skip + " or test_to_numpy" %}                   # [py<39]
 
 # ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
-{% set tests_to_skip = tests_to_skip + " or "test_date_max_2d" %}               # [s390x]
-{% set tests_to_skip = tests_to_skip + " or "test_date_min_2d" %}               # [s390x]
-{% set tests_to_skip = tests_to_skip + " or "test_date_median_2d" %}            # [s390x]
+{% set tests_to_skip = tests_to_skip + "test_date_max_2d" %}                    # [s390x]
+{% set tests_to_skip = tests_to_skip + " or test_date_min_2d" %}                # [s390x]
+{% set tests_to_skip = tests_to_skip + " or test_date_median_2d" %}             # [s390x]
 # AssertionError: assert array(...)
-{% set tests_to_skip = tests_to_skip + " or "test_asdatetime %}                 # [s390x]
+{% set tests_to_skip = tests_to_skip + " or test_asdatetime" %}                 # [s390x]
 
 
 # pandas 2.0.3 (last py38 build) really doesn't like the JSON

--- a/recipe/patches/0001-remove-test-against-base.BaseReduceTests.patch
+++ b/recipe/patches/0001-remove-test-against-base.BaseReduceTests.patch
@@ -1,0 +1,27 @@
+From f655709f62c23d7651ec3d444a6d18f7e4cffe88 Mon Sep 17 00:00:00 2001
+From: Ian Fitchet <ifitchet@anaconda.com>
+Date: Mon, 16 Sep 2024 16:42:01 +0100
+Subject: [PATCH] remove test against base.BaseReduceTests
+
+---
+ tests/compliance/json/test_json_compliance.py | 4 ----
+ 1 file changed, 4 deletions(-)
+
+diff --git a/tests/compliance/json/test_json_compliance.py b/tests/compliance/json/test_json_compliance.py
+index 18610a0..61348d0 100644
+--- a/tests/compliance/json/test_json_compliance.py
++++ b/tests/compliance/json/test_json_compliance.py
+@@ -238,10 +238,6 @@ class TestJSONArrayPrinting(base.BasePrintingTests):
+     pass
+ 
+ 
+-class TestJSONArrayReduce(base.BaseReduceTests):
+-    pass
+-
+-
+ class TestJSONArrayReshaping(base.BaseReshapingTests):
+     @pytest.mark.skip(reason="2D support not implemented for JSONArray")
+     def test_transpose(self, data):
+-- 
+2.39.3 (Apple Git-146)
+

